### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by m-samiec**
Small cleanup of repo.

@davedolben @improbable-valentyn

### CODE REVIEW

File: LaunchClient.bat [hunk 1]
```diff
@@ -1,3 +1,3 @@
 @echo off
 call "%~dp0ProjectPaths.bat"
-"%UNREAL_HOME%\Engine\Binaries\Win64\UE4Editor.exe" "%~dp0%PROJECT_PATH%\%GAME_NAME%.uproject" -game -log -workerType UnrealClient -stdout -nowrite -unattended -nologtimes -nopause -noin -messaging -NoVerifyGC -windowed -ResX=800 -ResY=600
+"%UNREAL_HOME%\Engine\Binaries\Win64\UE4Editor.exe" "%~dp0%PROJECT_PATH%\%GAME_NAME%.uproject" 127.0.0.1 -game -log -workerType UnrealClient -stdout -nowrite -unattended -nologtimes -nopause -noin -messaging -NoVerifyGC -windowed -ResX=800 -ResY=600
```
Comment (davedolben) [inline LaunchClient.bat]: Don't think this is actually necessary, it should connect to localhost on startup unless you've passed something else in for `+receptionistHost`

File: LaunchSpatial.bat [hunk 2]
```diff
@@ -0,0 +1,13 @@
+@echo off
```
Comment (davedolben) [inline LaunchSpatial.bat]: Cool, should probably put this in StarterProject and TestSuite too if it's not already there

File: LaunchClient.bat (hunk 1 above)
Comment (m-samiec) [reply LaunchClient.bat]: Yep, this has changed with the recent Server Travel PR, slightly jumping the gun with this fix.

File: LaunchSpatial.bat (hunk 2 above)
Comment (m-samiec) [reply LaunchSpatial.bat]: They've already got this, it failed to make it's way into TPS when it was in limbo.

**Pull request merged**